src/Scenario/InfoClientScenario.php

```diff
@@ -13,20 +14,39 @@ abstract class InfoClientScenario implements ScenarioInterface
     protected $infoClient;
     /** @var float */
     private $timeout = 3.0;
+    /** @var string */
+    private $authKey;
 
+    /**
+     * @param ClientGeneratorInterface|null $clientGenerator
+     *
+     * @throws TGException
+     */
     public function __construct(ClientGeneratorInterface $clientGenerator = null)
     {
         if (!$clientGenerator) {
             $clientGenerator = new ClientGenerator();
         }
         $this->infoClient = $clientGenerator->getInfoClient();
+        $this->authKey = $clientGenerator->getAuthKey();
     }
 
     public function setTimeout(float $timeout): void
     {
         $this->timeout = $timeout;
     }
 
+    /**
+     * @throws TGException
+     */
+    protected function infoLogin(): void
+    {
+        $authKey = AuthKeyCreator::createFromString($this->authKey);
+        if (!$this->infoClient->isLoggedIn()) {
+            $this->infoClient->login($authKey);
+        }
+    }
+
```

Comment:
Get rid of this method - "login" only should exists

Reply:
Fixed, renamed to `login`

Reply:
Renamed to `login`.